cupy/testing/_helper.py

```diff
@@ -226,7 +227,7 @@ def _make_positive_mask(self, impl, args, kw, name, sp_name, scipy_name):
 
 
 def _contains_signed_and_unsigned(kw):
-    vs = set(kw.values())
+    vs = set(filter(lambda x: isinstance(x, abc.Hashable), kw.values()))
```

Comment:
Could you add a comment to the function?  I suppose the function is intended to catch dtypes from `for_*_dtypes` decorators, and it was assumed that dtype should not be passed in other ways.

Is this better than `list(kw.values())`?  I would prefer `set` if the next line were `return vs.intersection(_unsigned_dtypes) and vs.intersection(_float_dtypes + _signed_dtypes)`.

Reply:
I made this change because kwargs as a dict was passed in `kw` and it was not hashable. Now I'm going to follow https://github.com/cupy/cupy/pull/5337/files#r658514045, so we don't need it anymore.

Reply:
(And, yes, using `list` would be enough even when we need some changes here)